tests/modules/datatype.c

```diff
@@ -113,6 +115,27 @@ static int datatype_restore(RedisModuleCtx *ctx, RedisModuleString **argv, int a
     return REDISMODULE_OK;
 }
 
+static int datatype_restore_encver(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+
+    if (argc != 3) {
+        RedisModule_WrongArity(ctx);
+        return REDISMODULE_OK;
+    }
+
+    DataType *dt = RedisModule_LoadDataTypeFromStringEncver(argv[2], datatype, DATATYPE_ENC_VER);
+    if (!dt) {
+        RedisModule_ReplyWithError(ctx, "Invalid data");
+        return REDISMODULE_OK;
+    }
+
+    RedisModuleKey *key = RedisModule_OpenKey(ctx, argv[1], REDISMODULE_WRITE);
+    RedisModule_ModuleTypeSetValue(key, datatype, dt);
+    RedisModule_CloseKey(key);
+    RedisModule_ReplyWithSimpleString(ctx, "OK");
+
+    return REDISMODULE_OK;
+}
```

Comment:
i'm not sure we really need to keep the old test for `RedisModule_LoadDataTypeFromStringEncver` and introduce a new one.
i.e. since this is just a wrapper, it is enough to test the more advanced variant and not test the one with missing argument.

Reply:
we can however improve the test and check that the provided `encver` is really propagated to the module during decoding (e.g. pass `4` and assert for it)